ckan/views/admin.py

```diff
@@ -143,26 +144,63 @@ class TrashView(MethodView):
     def __init__(self):
         self.deleted_packages = model.Session.query(
             model.Package).filter_by(state=model.State.DELETED)
+        self.deleted_orgs = model.Session.query(model.Group).filter_by(
+            state=model.State.DELETED, is_organization=True)
+        self.deleted_groups = model.Session.query(model.Group).filter_by(
+            state=model.State.DELETED, is_organization=False)
+
+        self.data_type = {
```

Comment:
You mean, `data_type` stores not the actual type of data, but rather a collection of deleted items?